boards/arduino/opta/Kconfig.defconfig

```diff
@@ -11,34 +11,6 @@ config NET_L2_ETHERNET
 
 endif # NETWORKING
 
-if USB_DEVICE_STACK
-
-config USB_DEVICE_PRODUCT
-	default "Arduino Opta"
-
-config USB_DEVICE_PID
-	default 0x0164
-
-config USB_DEVICE_VID
-	default 0x35d1
-
-config USB_DEVICE_INITIALIZE_AT_BOOT
-	default y
-
-if LOG
-
-# Logger cannot use itself to log
-choice USB_CDC_ACM_LOG_LEVEL_CHOICE
-	default USB_CDC_ACM_LOG_LEVEL_OFF
-endchoice
-
-# Set USB log level to error only
-choice USB_DEVICE_LOG_LEVEL_CHOICE
-	default USB_DEVICE_LOG_LEVEL_ERR
-endchoice
-
-endif # LOG
-
-endif # USB_DEVICE_STACK
+source "${ZEPHYR_BASE}/boards/common/Kconfig.cdc_acm_serial"
```

Comment:
Nitpick:
Harmonize path between Kconfig and dts:
`source "../boards/common/Kconfig.cdc_acm_serial"`
`#include <../boards/common/cdc_acm_serial.dts>`
or
`source "${ZEPHYR_BASE}/boards/common/Kconfig.cdc_acm_serial"`
`#include <${ZEPHYR_BASE}/boards/common/cdc_acm_serial.dts>`

Reply:
${ZEPHYR_BASE} cannot be resolved in dt.

Reply:
Can we then use `../boards/common` for Kconfig ?